build.sh

```diff
@@ -8,10 +8,22 @@ if [ -n "${INCLUDE_TESTS}" ]; then
     cmake_args='-DCMAKE_BUILD_TYPE=RelWithDebInfo -DINCLUDE_TESTS=1 ..'
 fi
 
+## Build the cpplite lib first
+#echo "Building the cpplite lib"
+mkdir cpplite/build.release
+cd cpplite/build.release
+cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_ADLS=ON -DUSE_OPENSSL=OFF
```

Comment:
will this work for blob storage without ADLS?

Reply:
This is bit hard to explain but cpplite does not have a composite lib that includes both block-blob and adls. When we build cpplite it actually creates two independent lib one for each. Now to build adls lib we need to pass these parameters. So BlockBlob lib is built by-default and BUILD_ADLS directs the cmake to build the ADLS part as well. As we need both have made these changes. OPENSSL we did not support earlier and enabling that was giving compilation failures so have turned this off. This command will build both the libs and keep them in biuld.release under cpplite folder (just the standard way of building cpplite, as mentioned in their README)